assets/scss/learndash-styling.scss

```diff
@@ -189,6 +190,9 @@
 
 	.ld-item-list-item {
 	  .ld-item-list-item-preview {
+	    .ld-item-title {
+			line-height: 2.25rem;
```

Comment:
why the hard coded value?

Reply:
do we have a soft coded equivalent?

Reply:
https://github.com/nhsuk/nhsuk-frontend/blob/master/packages/core/settings/_typography.scss
$nhsuk-typography-scale ...?